src/Common/src/TypeSystem/Ecma/EcmaModule.CodeGen.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+using System.Reflection.PortableExecutable;
+
+namespace Internal.TypeSystem.Ecma
+{
+    partial class EcmaModule
+    {
```

Comment:
True. It's a vestige from when I thought we would want to expose this on `ModuleDesc`. But then I realized we'll probably never do codegen from non-EcmaModule modules.